src/components/views/about-us/who-we-are/who-we-are.component.tsx

```diff
@@ -0,0 +1,42 @@
+import styles from "./who-we-are.module.scss";
+import { useTranslation } from "next-i18next";
+import { Container, Stack } from "@design-system/layout/utilities";
+import PunctualityIcon from "../../../shared/icons/punctuality.icon";
+import SecurityIcon from "../../../shared/icons/security.icon";
+import PartnershipIcon from "../../../shared/icons/partnership.icon";
+import StarsIcon from "../../../shared/icons/stars.icon";
+
+function WhoWeAre() {
+  const { t } = useTranslation("common");
+  return (
+    <div className={styles.wrapper}>
+      <Container>
+        <Stack className={styles.content}>
```

Comment:
<Stack> używaj jak chcesz żeby flexbox miał flex-direction:column
<FlexComponent> używaj jak chcesz żeby flexbox miał flex-direction: row

czyli podmień na FlexComponent i usuń tam w cssie to "flex-direction:row"